recipes/sleapyfaces/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "sleapyfaces" %}
+{% set version = "1.2.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sleapyfaces-{{ version }}.tar.gz
+  sha256: 5615177d7808ed70c3964639b7bda9e69a237f5ec19f4a692dc612cdebdad48f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - pip
+  run:
+    - python >=3.7
+    - click
+    - pytest
+    - pytest-cov
+    - pandas
+    - numpy
+    - scipy
+    - scikit-learn
+    - pytables
+    - ffmpeg-python
+    - h5py
+    - python-rapidjson
+    - matplotlib-base
+    - statsmodels
+    - seaborn
+    - geomstats
+    - plotly
+    - pathvalidate
+    - more-itertools
+    - python-configuration
+
+test:
+  imports:
+    - sleapyfaces
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/sleapyfaces/
+  summary: A package for extracting facial expressions from SLEAP analyses
+  dev_url: https://github.com/annie444/sleapyfaces
```

Comment:
Is this correct? That seems to be a fork.

Reply:
Yes, I am the maintainer and this is the official version. However, due to IP requirements, the repo has to be owned by my former employer (Tyelab) even though they are not maintaining it.

Reply:
Note that all of the commits on their repo are just PRs from me